tests/bug-reports/RemoveUnusedTypars_A.fs

```diff
@@ -0,0 +1,11 @@
+#light "off"
```

Comment:
This file shouldn't be checked in, and should be added to the gitignore.

Reply:
I've removed both RemoveUnusedTypars_A.fs and RemoveUnusedTypars_B.fs, but I'm not sure about adding those particular files to gitignore. In general it feels like any *.fs files could be ignored inside this directory, but there are no entries for any there. What do you think?

Reply:
> In general it feels like any *.fs files could be ignored inside this directory,

Indeed :), that'd be fine, or even better to just move these files away into `tests/` somewhere. Perhaps the machinery in `tests/machine_integers` helps (it extracts, runs, and verifies expected output; it should be easy to also compare the generated code).

Reply:
The make targets used for those tests seem to be fairly self-contained (unless I'm missing some make trickery there) so I'm fine with either solution as long as they are run as part of the CI :) @nikswamy Any preferences here?